west.yml

```diff
@@ -28,6 +28,10 @@ manifest:
     - name: canopennode
       revision: 1052dae561497bef901f931ef75e117c9224aecd
       path: modules/lib/canopennode
+    - name: chre
+      url: https://android.googlesource.com/platform/system/chre
```

Comment:
if module is approved we'll need a Zephyr fork of this.

Reply:
Done